### PR TITLE
Fix datatable search

### PR DESCRIPTION
**Pull request by axnish27**
Search is working now please check #20  @Raza9798

### CODE REVIEW

**Comment by axnish27** [comment]
@Raza9798  #20 please check code redundancy removed completely

File: app/Http/Controllers/EmployeeController.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 use App\Models\Employee;
+use Dotenv\Parser\Value;
```
Comment (Raza9798) [inline app/Http/Controllers/EmployeeController.php:5]: Please explain why have you used the `Parser` @axnish27

**Review by Raza9798 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/EmployeeController.php (hunk 1 above)
Comment (axnish27) [reply app/Http/Controllers/EmployeeController.php:5]: Sorry it was added by mistake...ill remove

Comment (axnish27) [reply app/Http/Controllers/EmployeeController.php]: @Raza9798  removed unnecessary files please check

**Review by Raza9798 — APPROVED** [review]

**Pull request merged**